src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -536,9 +551,14 @@ public void testEnrollLinesChecking() throws Exception {
 
         enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInputWithComment;
 
-        invalidInfo = getInvalidityInfoInEnrollLines(enrollLines, courseId);
+        invalidInfoString = null;
+        try {
+            studentsLogic.createAndValidateStudents(enrollLines, courseId);
+        } catch (EnrollException e) {
+            invalidInfoString = e.getMessage();
+        }
 
-        assertEquals(0, invalidInfo.size());
+        assertNull(invalidInfoString);
```

Comment:
I have changed it to just call 
    studentsLogic.createAndValidateStudents(enrollLines, courseId);
and added description on why no checking is done. I do not reuse getExceptionMessageOnCreatingStudentsList here as this does not throw exception.